Quandl.Excel.UDF.Functions/Helpers/Tools.cs

```diff
@@ -162,7 +162,15 @@ public static List<object> GetValuesFromCellRange(ExcelReference excelReference)
 
         public static string GetValueFromSingleCell(ExcelReference excelReference)
         {
-            return excelReference.GetValue().ToString();
+            // This is a very basic way of detecting date formatted cell. Checking for the existence of a y as most date formats will contain at least one 'y' for the year.
```

Comment:
I looked into that and seemed to work for me in most cases. Do you happen to know a format where y should not be case sensitive?